src/js/disability-benefits/components/FeaturesWarning.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 export default function FeaturesWarning() {
   return (
     <div className="claims-sidebar-box">
-      <h4 className="claims-sidebar-header">Other claim services</h4>
+      <h4 className="highlight claims-sidebar-header">Other claim services</h4>
```

Comment:
Updated to use `claim-file-border claim-h2` classes and `h2` tag for consistency, but this may need to be revisited. Why is the color for `claim-file-border` different from `highlight`?